More FAQs (#70)
packit-service/packit.dev: content/faq/_index.md

In case of any other questions, feel free to contact us:
**user-cont-team@redhat.com**.

### Why do I have to maintain .packit.yaml and a spec file upstream?
[We are working](https://github.com/packit-service/packit/issues/574) on simplifying the `.packit.yaml` so it's as small as possible.

> We will also handle all potentialy backward incompatible changes of `.packit.yaml`.

Reviewer: ```suggestion
We will also handle all potentially backwards incompatible changes of `.packit.yaml`.
```
my gut tells me it's usually spelled like this

Reviewer 2: ehm, no :) https://www.quickanddirtytips.com/education/grammar/backward-versus-backwards

Reviewer: Now I'm even more confused.

> `Spec` file can be downloaded (see specific question below) from [Fedora Pagure](https://src.fedoraproject.org).

Reviewer: ```suggestion
`Spec` file can be downloaded (see specific question below) from [Fedora Pagure](https://src.fedoraproject.org) instead of having it included in the upstream repository.
```

### But what are the benefits?
Packit makes it trivial to run your project as part of an OS.

> It provides feedback to upstream project at the time when the changes are being developed.

Reviewer: ```suggestion
It provides feedback to your project at the time when the changes are being developed so you can fix incompatible code when you are working on it, not when it's already released.
```

> With it the project gets (build/tests) results right away.

Reviewer: ```suggestion
When you push commits to a pull request, you'll get RPM build and test results right away.
```

### Why Fedora?

> We've started with Fedora because we work for Red Hat.

Reviewer: ```suggestion
We've started with Fedora Linux because we work for Red Hat and we ❤ Fedora.
```

### How is Packit different from other services?
Packit connects the existing services ([Copr](https://copr.fedorainfracloud.org), [Pagure](https://src.fedoraproject.org), [Koji](https://koji.fedoraproject.org), [Bodhi](https://bodhi.fedoraproject.org)) together.

### Can we use Packit with Gitlab?

> Packit service is closely tied to Github because most of the projects are hosted on Github.

Reviewer: ```suggestion
Packit service is closely tied to GitHub right now because most of the projects are hosted on GitHub.
```
